RCPTT_Tests/DAETests/SettingTcbFileOnlyActionedAfterApplyChanges.test

```diff
@@ -0,0 +1,66 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Element-Name: SettingTcbFileOnlyActionedAfterApplyChanges
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _lwJiAFDoEee5PKfRF9iJJw
+Runtime-Version: 2.0.1.201508250612
+Save-Time: 6/15/17 11:59 AM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+proc "reset_tcb_file" {
+	// Set tcb to ""
+	get-view "DAE Control Program" | get-group "Time Channel Settings" | get-combo -after [get-label "Change:"] | set_combo_by_index 0
+	get-view "DAE Control Program" | get-button "Apply Changes" | click
+	wait_for_dae_update_dialog_to_close 
+}
+
+assert_instrument_is_setup
+switch_to_dae_view
+
+with [get-view "DAE Control Program"] {
+	// Move to Time Channels tab
+	get-tab-folder | get-tab-item "Experiment Setup" | click
+    get-tab-folder -after [get-label "DAE Control Program"] -index 1 | get-tab-item "Time Channels" | click
+    
+    // Switch to "Use TCB File" and set tcb to blank
+    get-view "DAE Control Program" | get-button "Use TCB File" | click
+    reset_tcb_file
+    
+    // Set "Change" to TCB file 1
+    get-view "DAE Control Program" | get-group "Time Channel Settings" | get-combo -after [get-label "Change:"] | set_combo_by_index 1
+    // Check "Change" was set 
+	get-view "DAE Control Program" | get-group "Time Channel Settings" | get-combo -after [get-label "Change:"] 
+    | get-property selection | contains "RCPTT_TCB_1.dat" | verify-true
```

Comment:
Test fails here for me. I think you need to update the CleanIBEXServerFile context. Although you've added the files to the repo, they don't automatically get picked up by the context